hail_search/test_search.py

```diff
@@ -1,7 +1,286 @@
 from aiohttp.test_utils import AioHTTPTestCase
 
+from hail_search.test_utils import get_hail_search_body, FAMILY_2_VARIANT_SAMPLE_DATA, FAMILY_2_MISSING_SAMPLE_DATA
 from hail_search.web_app import init_web_app
 
+VARIANT_ID_1 = '1-10439-AC-A'
+VARIANT1 = {
```

Comment:
Should we re-use the test data constants directly from the ES search? The pros include less typing and fewer mistakes. One of the main cons is coupling the tests with the ES searches.